tests/test_functions.py

```diff
@@ -98,6 +102,33 @@ def test_reverse_hash():
     assert s == "GGGG"
 
 
+def test_reverse_hash_longs():
+    # test explicitly with long integers, only needed for python2
+    # the builtin `long` exists in the global scope only
+    global long # pylint: disable=global-variable-undefined
+    if sys.version_info > (3,):
+        long = int
+
+    s = khmer.reverse_hash(long(0), 4)
+    assert s == "AAAA"
+
+    s = khmer.reverse_hash(long(85), 4)
+    assert s == "TTTT"
+
+    s = khmer.reverse_hash(long(170), 4)
+    assert s == "CCCC"
+
+    s = khmer.reverse_hash(long(255), 4)
+    assert s == "GGGG"
+
+
+def test_reverse_hash_raises():
+    with pytest.raises(TypeError) as excinfo:
+        khmer.reverse_hash('2345', 4)
+
+    assert 'must be int' in str(excinfo.value)
```

Comment:
Looks like this is due to slightly different error messages. Made the check less stringent.